pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -291,6 +290,10 @@ func (b *awsElasticBlockStoreBuilder) SetUpAt(dir string) error {
 		return err
 	}
 
+	if !b.readOnly {
```

Comment:
Ah I see... Yeah I don't know if we can assume that the default ownership will always be 0. In addition (for better or for worse) the ownership management also sets `setgid`, and that is less likely to be the default so we might as well run it.